src/workers/python/init.py

```diff
@@ -39,10 +101,19 @@ def show():
 
         matplotlib.pyplot.show = show
 
+    def pre_run(self, source_code, mode="exec", top_level_await=False):
+        self.override_globals()
+        return super().pre_run(source_code, mode=mode, top_level_await=top_level_await)
+
+    def execute(self, code_obj, source_code, mode=None):  # noqa
+        return eval(code_obj, self.console.locals)  # noqa
```

Comment:
What's this about?

Reply:
PythonRunner only overrides methods when execute is called. I moved that to override_builtins to ensure they are overwritten early enough so that everything is always captured. I needed it for something Pdb-related, but I also am not sure why python_runner only applies the patches at execute time (and thus every time code is executed).

Reply:
Should I make a PR for those patches (together with the file-creation) in the python_repository or is this something you want to do yourself?

Reply:
I'd appreciante a PR :)

Maybe PyodideRunner could also optionally `install_imports` in pre_run/execute.

Reply:
I'll move the overriding there as well, and allow users to toggle it? (as discussed above) I'll await another collaborator invite then?

Reply:
> 
> Maybe PyodideRunner could also optionally `install_imports` in pre_run/execute.

This installation is async, so pre_run or execute is not possible. Should I add it to run_async instead?

Reply:
Making other methods async might cause trouble in your other packages. However, if we are making such changes, we might want to make the source_code also a member variable, so it doesn't need to be passed along to every method?

Reply:
I forgot about the asyncness. Forget that part.

Reply:
Having this method in python_runner might be nice, but i'll keep it within Papyros for now then?